resources/views/frontend/layouts/partials/sitemaptop.blade.php

```diff
@@ -0,0 +1,14 @@
+<div class="siteMapTop">
+  <div class="siteMapTop_container">
+    <div class="fright siteMapTop_logout2">
+      <a href="/" onClick="accessToLogin(this); return false;" class="siteMapTop_color">
```

Comment:
[rank A] please don't use js like that. We need define it as jQuery click function.